mineinabyss-features/src/main/kotlin/com/mineinabyss/features/configs/ConfigReloadsFeature.kt

```diff
@@ -0,0 +1,44 @@
+package com.mineinabyss.features.configs
+
+import com.mineinabyss.idofront.commands.arguments.stringArg
+import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.success
+import com.mineinabyss.mineinabyss.core.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+
+@Serializable
+@SerialName("config_reloads")
+class ConfigReloadsFeature : AbyssFeature {
+    @Transient
+    private val configs = abyssFeatures.features.filterIsInstance<Configurable<*>>().associateBy { it.configManager.name }
```

Comment:
this tries to access Feature-context before it has been registered
this should probably be a lazy or something